src/dnf-plugins/subscription-manager.py

```diff
@@ -54,9 +54,9 @@ def __init__(self, base, cli):
         super(SubscriptionManager, self).__init__(base, cli)
         self.base = base
         self.cli = cli
-        self.config()
+        self._config()
 
-    def config(self):
+    def _config(self):
```

Comment:
Won't this cause the plugin not to run on any dnf hook?

Reply:
The `_config()` will be triggered. The `__init__()` is sort of hook in this case. This 'hook' is triggered as the first.